tools/snitch/snitcher/src/lib.rs

```diff
@@ -452,66 +427,51 @@ where
     })
 }
 
-#[cfg(test)]
-mod tests {
-    extern crate env_logger;
-
-    use super::*;
-
-    #[test]
-    fn module_logs() {
-        env_logger::init();
-
-        let settings = Settings::default().merge_env().unwrap();
-        let tasks = get_module_logs(&settings)
-            .map(|_| ())
-            .map_err(|err| panic!("ERROR: {:?}", err));
-        tokio::run(tasks);
-    }
+pub trait UrlExt {
```

Comment:
We should be able to add a reference to the `edgelet-core` crate instead of copying this code.